documentation/adr/0024-page-to-single-file.md

```diff
@@ -0,0 +1,47 @@
+[<-previous](0023-error-format.md)
+next->
+
+# 24 - Page PixelData as single file
+
+Date: 06-10-2022
+
+## Status
+
+Accepted
+
+## Context
+
+When performing file-backed operations, the [current implementation](0018-algorithm-migration-to-paged-files.md) of
+`PixelData` uses multiple files to store the temporary files and subsequently rebuilds them upon completion of the
+operation. While this might be efficient for partial (applied to subsections of data) access patterns, this is not the
+case for either full-sequential (applied to whole data [unary/binary ops]) or random-access (applied with variable
```

Comment:
For random-access this is fine. For full-sequential, I thought we said that the multiple file storage was fine as each file separately would undergo the operation.